### PR TITLE
Remove code for running on Ruby on Windows

### PR DESCRIPTION
**Pull request by Fryguy**
@roliveri Please review.

### CODE REVIEW

File: lib/disk/modules/MiqLargeFile.rb [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-  end
+if Sys::Platform::IMPL == :linux && MiqSystem.arch == :x86_64
+  require 'linux_block_device'
+  require 'disk/modules/RawBlockIO'
 end
 
 module MiqLargeFile
   def self.open(file_name, flags)
-    case Sys::Platform::OS
-    when :unix
```
Comment (Fryguy) [inline lib/disk/modules/MiqLargeFile.rb]: Note that `Sys::Platform::OS == :unix` on both MacOS and Linux, but not Windows

**Comment by Fryguy** [comment]
cc @jerryk55 
See also https://github.com/ManageIQ/manageiq-smartstate/pull/15

**Comment by Fryguy** [comment]
Also added some cleanup so that the sys-uname gem is properly required (including creating a method in test_helper so we don't have to require it separately in multiple test files).

**Comment by chessbyte** [comment]
@Fryguy @roliveri we should remove the windows agent support.  just cluttering the code base.

**Comment by Fryguy** [comment]
@roliveri Rebased

**Pull request merged**